Formula/l/lexido.rb

```diff
@@ -0,0 +1,24 @@
+require "English"
+class Lexido < Formula
+  desc "Innovative assistant for the command-line"
+  homepage "https://github.com/micr0-dev/lexido"
+  url "https://github.com/micr0-dev/lexido/archive/refs/tags/v1.3.1.tar.gz"
+  sha256 "8d05016b392fa43a33d6989c1f6e568f0d8b5d8893faf8622d02d36fd2d5e9db"
+  license "AGPL-3.0-or-later"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    # Set a fake API key for testing purposes
+    ENV["GOOGLE_AI_KEY"] = "12309863243987"
+
+    # Run the `lexido` command
+    shell_output("#{bin}/lexido 2>&1", 1).strip
```

Comment:
If it's an error we should try to match something from the shell output.

```suggestion
    assert_match "<what error message does it produce?>", shell_output("#{bin}/lexido 2>&1", 1).strip
```

Reply:
```suggestion
    assert_match "error details: googleapi: got HTTP response code 400", shell_output("#{bin}/lexido 2>&1", 1).strip
```